src/main/java/com/perficient/movies/model/dtos/ActorDto.java

```diff
@@ -20,4 +20,5 @@
     private int startYear;
```

Comment:
SonarLint Issues

Reply:
SonarLint didn't tell me anything about it. Can you tell me what is pls? :(